pd/src/consensus/worker.rs

```diff
@@ -167,7 +167,20 @@ impl Worker {
             .expect("can parse app_state in genesis file");
 
         // Begin new sidecar code
+
+        // We want to write the genesis state as version 0 of the tree, so we
+        // need to initialize the write overlay with the PRE_GENESIS_VERSION.
+        // To do this, it's easiest to just overwrite the app here.
+        self.app = App::new(Arc::new(Mutex::new(WriteOverlay::new(
+            self.storage.clone(),
+            // Some kind of visibility issue on associated consts?
+            //WriteOverlay::PRE_GENESIS_VERSION,
```

Comment:
I'm not sure why, but using `WriteOverlay::PRE_GENESIS_VERSION` here doesn't work, because rustc thinks that the associated constant isn't there.  For now, this just puts in the value directly.